sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/MessageUtils.java

```diff
@@ -71,4 +72,8 @@ public static UUID convertDotNetBytesToUUID(byte[] dotNetBytes) {
         long leastSignificantBits = buffer.getLong();
         return new UUID(mostSignificantBits, leastSignificantBits);
     }
+    // Pass little less than client timeout to the server so client doesn't time out before server times out
+    public static Duration adjustServerTimeout(Duration clientTimeout) {
+        return clientTimeout.minusMillis(200);
```

Comment:
Could this cause an issue if the `clientTimeout` is less than `200` milliseconds?

Reply:
This is how it is setup in track1. So I think it is safe to take this logic.